apps/system/js/attention_window_manager.js

```diff
@@ -45,6 +45,14 @@
       return this._topMostWindow;
     },
 
+    showCallscreenWindow: function attwm_getCallscreenWindow() {
+      var callscreenWindow = this._instances.find(function(aw) {
+        return aw.isCallscreenWindow;
+      });
+
+      callscreenWindow && callscreenWindow.requestOpen();
+    },
```

Comment:
I'll test it then, that would be very nice and I'd get rid of the related system-app hacks.